app/views/eventDetails/regTypes.html

```diff
@@ -230,40 +230,70 @@ <h4 translate>Group Registration</h4>
+                  ng-if="conference.cruEvent && !(conference.businessUnit || conference.operatingUnit || conference.department || conference.projectId)"
+                >
+                </i>
+              </h4>
               <div class="checkbox">
                 <label>
-                  <input type="checkbox" ng-model="type.acceptCreditCards" />
+                  <input
+                    type="checkbox"
+                    ng-model="type.acceptCreditCards"
+                    ng-disabled="!type.acceptCreditCards && (!conference.paymentGatewayId || (conference.cruEvent && !(conference.businessUnit || conference.operatingUnit || conference.department || conference.projectId)))"
+                  />
                   <translate>Accept Credit Cards</translate>
                 </label>
                 <i
-                  class="question-popover fa fa-question-circle"
-                  uib-popover="{{'Enter credit card processing details under the Payment Options tab.' | translate}}"
+                  class="warning-popover fa fa-exclamation-triangle"
+                  uib-popover="{{'Enter credit card processing details under the Payment Options tab to enable this option.' | translate}}"
                   popover-trigger="mouseenter"
+                  ng-if="!conference.paymentGatewayId"
                 >
                 </i>
               </div>
               <div class="checkbox">
                 <label>
-                  <input type="checkbox" ng-model="type.acceptChecks" />
+                  <input
+                    type="checkbox"
+                    ng-model="type.acceptChecks"
+                    ng-disabled="!type.acceptChecks && conference.cruEvent && !(conference.businessUnit && conference.operatingUnit && conference.department && conference.projectId)"
+                  />
                   <translate>Accept Checks</translate>
                 </label>
               </div>
               <div class="checkbox">
                 <label>
-                  <input type="checkbox" ng-model="type.acceptTransfers" />
+                  <input
+                    type="checkbox"
+                    ng-model="type.acceptTransfers"
+                    ng-disabled="!type.acceptTransfers && conference.cruEvent && !(conference.businessUnit && conference.operatingUnit && conference.department && conference.projectId)"
+                  />
                   <translate>Accept Account Transfers</translate>
                 </label>
               </div>
               <div class="checkbox">
                 <label>
-                  <input type="checkbox" ng-model="type.acceptScholarships" />
+                  <input
+                    type="checkbox"
+                    ng-model="type.acceptScholarships"
+                    ng-disabled="!type.acceptScholarships && conference.cruEvent && !(conference.businessUnit && conference.operatingUnit && conference.department && conference.projectId)"
```

Comment:
Do you think since these same four variables are used over and over again it would be better to define them as a single variable instead? Or maybe even a function?
```
$scope.paymentTypeIsDisabled = paymentType => {
      return (
        !paymentType &&
        conference.cruEvent &&
        !(
          conference.businessUnit &&
          conference.operatingUnit &&
          conference.department &&
          conference.projectId
        )
      );
    };
```

Reply:
AngularJS makes this too hard. But ya probably. I can do that.

Reply:
ReactJS rewrite being scheduled sometime in the mid term future.

Reply:
Note that not all of these chart field components need to be set in order for it to be considered a valid chart field. For some chart fields, it's possible for `projectId` to be blank. I don't know if there may be other chart fields where some other components might also be empty.

Ultimately, the ert-api will validate on conference save, so the save request will be denied if the chart fields are not valid and/or if the conference is including support for payments without any chart field set.

You can still attempt front end validation, if you deem that a better user experience (you'll still get some semi meaningful error message as response from ert-api, namely `Conference accepting any type of payment must provide chart field`) but at the very least, it cannot include requiring `projectId` be set.

Reply:
Looks like I updated one to consider it valid when any field was provided but I forgot to update the others. Guess that's another argument to write a function haha. Thanks for catching that.

Reply:
I've been informed that only Project ID would be _possible_ (some chart fields require it, some don't) to be left blank in a valid chart field.

Reply:
However, having said that, I'm now thinking that it should not be incumbent on the UI to understand valid PS Financials chart field validation rules. The ert-api doesn't know directly either, but must enquire of PS Financials in order to determine. 

Thoughts?

Reply:
Seems like we could either:
1. Require `businessUnit`, `operatingUnit`, and `department` to match PS Financials
1. Require `businessUnit`, `operatingUnit`, or `department` to just make sure they entered something

I'm assuming it'd be unlikely for these rules to change?

Reply:
ert-api calls out to PS Financials endpoint to validate. 
Option 2 seems sufficient.
I don't think the rules will change.